api/v1beta1/spannerautoscaler_types.go

```diff
@@ -108,6 +108,14 @@ type ScaleConfig struct {
 	// The cool down period between two consecutive scaledown operations. If this option is omitted, the value of the `--scale-down-interval` command line option is taken as the default value.
 	ScaledownInterval *metav1.Duration `json:"scaledownInterval,omitempty"`
 
+	// The maximum number of processing units which can be deleted in one scale-up operation. It can be a multiple of 100 for values < 1000, or a multiple of 1000 otherwise.
```

Comment:
```suggestion
	// The maximum number of processing units which can be added in one scale-up operation. It can be a multiple of 100 for values < 1000, or a multiple of 1000 otherwise.
```